lib/msal-electron-proof-of-concept/src/AuthCodeListener/AuthCodeListener.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * AuthCodeListener is the base class from which
+ * special CustomFileProtocol and HttpAuthCode inherit
+ * their structure and members.
+ */
+export abstract class AuthCodeListener {
```

Comment:
The reason this class is abstract is that there will be two types of listeners (custom protocol and http server). Custom protocol will be the default, but developers may want to switch to HTTP for preference.